mininet/netcfg.json

```diff
@@ -4,7 +4,10 @@
       "basic": {
         "managementAddress": "grpc://mininet:50001?device_id=1",
         "driver": "stratum-bmv2",
-        "pipeconf": "org.onosproject.ngsdn-tutorial"
+        "pipeconf": "org.onosproject.ngsdn-tutorial",
```

Comment:
I'd rather keep the netcfg simple, there's already enough information to consume. I haven't tried this tutorial with gui2, do you think it's ok to avoid pinning devices and hosts in position? e.g. if hosts or devices overlap, that is not good

Reply:
the reason I wanted to do this is to guarantee the layout is the same as your `topo.png` 
I think having free floating nodes is only distracting from the core content of what the student is trying to learn.

Reply:
If you want to discuss more, I am happy to receive your feedback on this. everything in GUI2 is open for improvement, as long as there is an interest in it.

Reply:
As I said, I'd prefer to reduce the amount of information in the netcfg, but I would not make a case for it. We can leave these changes as is. I haven't tried GUI2 with this topology without coordinates. With the old GUI, the topology graph was automatically arranged in a clear way that was easy enough to map to the one shown in `topo.png`.